README.md

```diff
@@ -58,8 +58,8 @@ The last command compiles the whole thing and copies the plugin into your local
 The next time you start Pidgin, or another libpurple client, you should be able to activate it in the "Plugins" window.
 
 ##### 2B. OR: Install from a community repo
-* Arch - AUR: https://aur.archlinux.org/packages/libpurple-lurch-git/
-* Fedora - COPR:  https://copr.fedorainfracloud.org/coprs/treba/pidgin-lurch/
+* Arch - AUR: [libpurple-lurch](https://aur.archlinux.org/packages/libpurple-lurch/) or [libpurple-lurch-git](https://aur.archlinux.org/packages/libpurple-lurch-git/)
```

Comment:
`libpurple-lurch` has been moved from AUR into core Arch so the first link went 404 by now. Could be fixed on the go or not. Also, there are potentially more repos/distros to list here (now or later), see https://repology.org/projects/?search=lurch